### PR TITLE
[3828] Autocomplete in checkout guest form

### PR DESCRIPTION
**Pull request by Dagnija-Viktorova**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3828

**Problem:**
* Browser doesn't offer to autofill a form in checkout

**In this PR:**
* Added autocomplete: "email" to CheckoutGuestForm component

### CODE REVIEW

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Comment by aleksandrakorolova** [comment]
Chrome: works
Safari: doesn't offer, as I understand it's expected
Opera and FF: don't offer too
Can you investigate this?
https://drive.google.com/file/d/1_DUAGEVKs9SkaI9eNlCi9hxxbViSnkjI/view?usp=sharing

**Comment by Dagnija-Viktorova** [comment]
I tried to fill the email form on Opera and it gave the option to use saved addresses although on first time it requires a couple of page refreshes to work, but I did an investigation for Firefox autocomplete feature and there's not a lot of information about how autofill works on Firefox since it's not even a globally available feature and is very inconsistent

What I found out is that it mostly doesn't work when it's only one field (not exclusive to email and also doesn't always work for full forms, for example, on zara.com), but adding another field (either by copying a field that does autofill or by creating your own field) does not solve the issue, but the requirements for form autofilling are very unclear. It's worth noting that only addresses from countries that already have autofill feature officially out (US and I assume Canada) will get offered.

In conclusion, I couldn't find a solution for email autocomplete on Firefox, and I have no more ideas on how to make the autocomplete cross-browser compatible

**Comment by aleksandrakorolova** [comment]
Ok, thank you for the investigation
